pkg/osvscanner/osvscanner.go

```diff
@@ -357,7 +381,25 @@ func scanGit(r reporter.Reporter, query *osv.BatchedQuery, repoDir string) error
 	}
 	r.PrintText(fmt.Sprintf("Scanning %s at commit %s\n", repoDir, commit))
 
-	return scanGitCommit(query, commit, repoDir)
+	err = scanGitCommit(query, commit, repoDir)
+	if err != nil {
+		return err
+	}
+
+	submodules, err := getSubmodules(repoDir)
+	if err != nil {
+		return err
+	}
+
+	for _, s := range submodules {
+		r.PrintText(fmt.Sprintf("Scanning submodule %s at commit %s\n", s.Path, s.Expected.String()))
+		err = scanGitCommit(query, s.Expected.String(), repoDir)
```

Comment:
Actually, can we set repoDir to the dir of the actual submodule? that way the table report points to the exact dir that contains the vuln.

Reply:
Done in 89c03a1